boards/openlabs-kw41z-mini-256kib/Makefile.dep

```diff
@@ -0,0 +1,2 @@
+USEMODULE += board
```

Comment:
I don't think this is necessary. `board` is a default module already.

Reply:
Indeed.